gossip/privdata/coordinator.go

```diff
@@ -407,63 +409,51 @@ func getTxInfoFromTransactionBytes(envBytes []byte) (*txInfo, error) {
 	txInfo := &txInfo{}
 	env, err := protoutil.GetEnvelopeFromBlock(envBytes)
 	if err != nil {
-		logger.Warningf("Invalid envelope: %s", err)
-		return nil, err
+		return nil, fmt.Errorf("invalid envelope: %s", err)
```

Comment:
There's a convention of using errors.Errorf from github.com/pkg/errors for this kind of error - any specific reason to use fmt here?

Reply:
I believe `github.com/pkg/errors` isn't supporting any new functionality as go2 errors design is in process. As such eventually the package will no longer be updated/supported. The standard now seems to be moving towards the standard go library pkg `errors` which does not have an `Errorf` formatting method and instead defers to `fmt.Errorf` for formatted errors.

Reply:
Ah thanks Danny - good to know!